src/AppInstallerRepositoryCore/Microsoft/PinningIndex.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "SQLiteWrapper.h"
+#include "Microsoft/Schema/IPinningIndex.h"
+#include "Microsoft/SQLiteStorageBase.h"
+#include "winget/Pin.h"
+#include <winget/ManagedFile.h>
+
+namespace AppInstaller::Repository::Microsoft
+{
+    struct PinningIndex : SQLiteStorageBase

```

Comment:
I didn't remember I had already added pins from manifest to the tracking index for `RequireExplicitUpgrade`... I need to figure out how to deal with those two at the same time or how to merge them.